src/stylesheets/components/global/textInput.scss

```diff
@@ -1,19 +1,24 @@
 .text-input {
-    width: 100%;
-    padding: 8px 0;
+	width: 100%;
+	padding: 8px 0;
 
-    &__label {
-        display: block;
-        margin-bottom: 10px;
-    }
-    &__input {
-        width: 100%;
-        background: $text-input-bg;
-        border-radius: 99px;
-        border: none;
-        outline: none;
-        padding: 10px 15px;
-        color: white;
-        font-size: 1.125em;
-    }
-}
\ No newline at end of file
+	&__label {
+		display: block;
+		margin-bottom: 10px;
+	}
+	&__input,
+	&__input--centered {
+		width: 100%;
+		background: $text-input-bg;
+		border-radius: 99px;
+		border: none;
+		outline: none;
+		padding: 10px 15px;
+		color: white;
+		font-size: 1.125em;
+	}
+
+	&__input--centered {
```

Comment:
Instead of having 2 separate ```__input--centered``` blocks, you can the above style block just for ```&__input```, and then nest a ```&--centered``` block inside it. Then, make the first style ```@extend .text-input``` to inherit all the relevant styles